src/index.ts

```diff
@@ -1,7 +1,9 @@
-import { readFileSync } from 'fs';
+import { readFileSync, readdirSync } from 'fs';
 import { resolve } from 'path';
 
 export function conventionalCommitsTypes(locale?: string) {
-  const fileName = locale ? `${locale}.json` : 'en.json';
+  const localizationFiles = readdirSync(resolve(__dirname, `../locale/`));
+  const fileName =
+    locale && localizationFiles.includes(`${locale}.json`) ? `${locale}.json` : 'en.json';
```

Comment:
```suggestion
  const fileName = localizationFiles.includes(`${locale}.json`) ? `${locale}.json` : 'en.json';
```
It can deal with the undefined and empty string input.